### ISSUE TITLE
[ENHANCEMENT] Installer prompt sequence in case of config file

### ISSUE THREAD

**Issue by prashantgnet** [harvester/harvester#3661]
**Is your enhancement related to a problem? Please describe.**
In case of deployment of harvester cluster using config file, user input prompt for config file must come up very early in sequence so that end user don't need to provide inputs manually which are already present in harvester config file.

**Describe the solution you'd like**
User input prompt must be first prompt in case of ISO based cluster deployment.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by cgahlon** [comment]
Agreed.  It would be great if an ISO based install ask for basic network info first, then the URL of the configuration manifest.  If a file URL is not supplied (or is missing necessary info) then proceed with additional prompts.

**Comment by bk201** [comment]
@prashantgnet Along with https://github.com/harvester/harvester/issues/3667, could you share why pxe boot installation doesn't work for you?

**Comment by prashantgnet** [comment]
Mentioned issue is for ISO based installer and not for pxe boot.

**Comment by gdttn** [comment]
Simply adding an installer argument for a config file would really help this use case - this was present in 0.x after all, unsure why it was removed.

(FWIW we can't use PXE because we have an established and slightly nonstandard PXE service which we use for the bulk of our machines; harvester is just a small part of our network, and ISO install over serial console is fast and repeatable by support staff).

**Comment by w13915984028** [comment]
This looks to be a further enhancement:

In the ISO installation, input the `harvester-configuration` file asap. and has an option to skip/continue all following input windows, thus it will be half-automated, without depending on a PXE server, but is still faster than `step-by-step`.

cc @bk201 @guangbochen
